perfkitbenchmarker/gcp/gce_virtual_machine.py

```diff
@@ -223,9 +226,18 @@ def AddMetadata(self, **kwargs):
     vm_util.IssueCommand(cmd)
 
 
-class ContainerizedGceVirtualMachine(GceVirtualMachine,
-                                     linux_vm.ContainerizedDebianMixin):
+class DebianContainerizedGceVirtualMachine(GceVirtualMachine,
```

Comment:
I don't really like inheriting from both ContainerizedMixin and DebianMixin since they are both derived from the same class... let me think about it for a while to see if there's a better way that doesn't require a bunch of extra work - If you can think of an alternative, that would be great